rules/crowdstrike_rules/crowdstrike_detection_passthrough.py

```diff
@@ -1,22 +1,22 @@
-from panther_base_helpers import crowdstrike_detection_alert_context
+from panther_base_helpers import crowdstrike_detection_alert_context, get_crowdstrike_field
 
 
 def rule(event):
-    return event.get("ExternalApiType") in "Event_DetectionSummaryEvent"
+    return get_crowdstrike_field(event, "ExternalApiType") in "Event_DetectionSummaryEvent"
```

Comment:
For Crowdstrike.FDREvent will still return the field value, after searching in `event`, right?

Reply:
uh, yes. What else? Maybe I'm not getting what you mean.

Reply:
So in some cases we won't find it because `Crowdstrike.FDREvent` contains all event types. Previously there was a guarantee that the field would exist due to the log type assignment. So I think `None in "Event_DetectionSummaryEvent"` will raise an exception?

Reply:
Converted to `==` with a default

Reply:
Resolving, as it's old and has been done.